common/control_files/contrail-vrouter-dpdk.ini

```diff
@@ -3,10 +3,8 @@ command=/usr/bin/contrail-vrouter-dpdk --no-daemon
 priority=410
 autostart=true
 killasgroup=true
-stopsignal=KILL
 stdout_capture_maxbytes=1MB
 redirect_stderr=true
 stdout_logfile=/var/log/contrail/contrail-vrouter-dpdk-stdout.log
-stderr_logfile=/dev/null
-startsecs=5
```

Comment:
1. The start time of vRuter/DPDK depends on huge pages available. So for some servers startsecs=5 just an unnecessary few sec delay, and for other it's even not enough to initialize the memory.
The default 1 second value seems quite reasonable to me.

2. vRouter/DPDK catches TERM and INT to shutdown gracefully. So the default TERM suits better than KILL we had previously.